src/components/SortBy.jsx

```diff
@@ -0,0 +1,21 @@
+import { useState } from "react"
+
+export function SortBy(){
+
+    const [currentItem,setCurrentItem]=useState()
+
+    const dropdownChange=(value) =>{
+        setCurrentItem(value);
+    }
+
+    return (
+            <section id='sort-by'>
+                <label htmlFor="sort-by">Sort By</label>
+                <select onChange={(event)=>dropdownChange(event.target.value)} 
```

Comment:
You could get away with just saying `(event) => setCurrentItem(event.target.value)`, no need for a separate function!